presto-tests/src/test/java/io/prestosql/memory/TestMemoryManager.java

```diff
@@ -379,19 +393,13 @@ public void testQueryMemoryPerNodeLimit()
         }
     }
 
-    public static DistributedQueryRunner createQueryRunner(Session session, Map<String, String> properties)
+    public static DistributedQueryRunner createQueryRunner(Session session, Map<String, String> extraProperties)
             throws Exception
     {
-        DistributedQueryRunner queryRunner = new DistributedQueryRunner(session, 2, properties);
```

Comment:
nit: you could move it to separate commit

Reply:
moved